app/components/ui/CalculatorInput.tsx

```diff
@@ -25,6 +25,7 @@ import { Label } from "@/components/ui/label";
 const CalculatorInput = () => {
   const [view, setView] = useState("form");
   const [data, setData] = useState<Household | null>(null);
+  const [formData, setFormData] = useState<formType | null>(null);
```

Comment:
I don't think we need to hold these values in state a second time - we can already access this with `methods.getValues()`

I'm not sure if `methods` is a standard naming convention - maybe we just call it `form`?